app/view_models/nsm/v1/core_cost_summary.rb

```diff
@@ -51,7 +51,7 @@ def show_allowed?
       end
 
       def any_changed?
-        [disbursements, letters_calls, work_items].any? { |rows| rows.any?(&:changed?) }
+        [disbursements, letters_calls, work_items].any? { |rows| rows.any?(&:changed?) } || additional_fees.changed?
```

Comment:
Again, this breaks the supposition that there could be multiple additional fees, by checking a single AdditionalFee model that knows it's a youth court fee. Slightly nicer if we could add an `additional_fees` array to the array on the left hand side of this line.